test/spec/unit/adapters/analytics/AnalyticsAdapter_spec.js

```diff
@@ -29,7 +29,6 @@ FEATURE: Analytics Adapters API
         const adapter = new AnalyticsAdapter(config);
         var spyTestGlobal = sinon.spy(window, config.global);
 
-        adaptermanager.registerAnalyticsAdapter(adapter, 'adapter');
```

Comment:
There is no consequence, which is why I removed them.  All the unit tests were using the `adapter` instance directly rather than getting it from the adapters registered with Prebid.js.  That's fine, but makes these `registerAnalyticsAdapter` lines irrelevant.